subsys/bluetooth/shell/bt.c

```diff
@@ -1313,10 +1390,58 @@ static int cmd_auth_passkey(const struct shell *shell,
 }
 #endif /* CONFIG_BT_SMP) || CONFIG_BT_BREDR */
 
+/* Dummy command to ensure that help is printed. */
+static int cmd_empty(const struct shell *shell, size_t argc, char *argv[])
+{
+	return 0;
+}
 
 #define HELP_NONE "[none]"
 #define HELP_ADDR_LE "<address: XX:XX:XX:XX:XX:XX> <type: (public|random)>"
 
+SHELL_CREATE_STATIC_SUBCMD_SET(sub_bt_advertise_on_cmds)
+{
+	/* Alphabetically sorted. */
+	SHELL_CMD_ARG(discov, NULL, "Discoverable",
+			cmd_advertise_conn_discov, 0, 0),
```

Comment:
Hmm, `0, 0` ? Are you sure about that?

Reply:
@Vudentz :
Values are a bit mixed right now:
- 0 for mandatory means argument means that check is disabled,
- 0 for optional means that it has 0 optional arguments

Do you have an idea how it could be unified?